spec/v3/services/env_var/delete_spec.rb

```diff
@@ -28,6 +28,6 @@
     end
 
     example { expect(last_response.status).to eq 200 }
-    example { pending 'should we return an empty body here?' }
+    example { expect(JSON.parse(last_response.body)["id"]).to eq(env_var[:id]) }
```

Comment:
I'm going to open a ticket to have all our DELETE's return and empty body instead of the deleted object and save this work for another PR